website/docs/components/form/super-select/partials/code/component-api.md

```diff
@@ -211,6 +218,13 @@ The default values of some [ember-power-select](https://ember-power-select.com/d
     <br><br>
     Note: It cannot be overridden.
   </C.Property>
+  <C.Property @name="searchFieldPosition" @type="string">
+    Determines whether the search field, when enabled, renders either within the before options content or in the “trigger” area.
+    <br><br>
+    Set to `"before-options"` to fix a `PowerSelect` accessibility issue. (Rendering the search input within the trigger area creates a nested interaction.)
```

Comment:
```suggestion
    Set to `"before-options"` to fix a `PowerSelect` accessibility issue.  Rendering the search input within the trigger is not compliant because it nests an interactive element within another interactive element.
```